tools/gohan_client_bash_completion.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+
+commands="list show create set delete"
+verbosity="0 1 2"
+outputFormat="json table"
+gohanCommandPos=1
+unnamedCommandWords="show set delete"
```

Comment:
These variables are leaked to the environment. Ideally, they shouldn't be visible outside the completion script.

Maybe there's a way to make them not visible. Or at least it's possible to add some underscores in the beginning of names.